src/coloring/acyclic_coloring.jl

```diff
@@ -30,7 +30,14 @@ function color_graph(g::Graphs.AbstractGraph, ::AcyclicColoring)
                 for x in outneighbors(g, w)
                     if color[x] != 0
                         if forbidden_colors[color[x]] != v
-                            prevent_cycle!(first_visit_to_tree,forbidden_colors,v, w, x, g, two_colored_forest,color)
+                            prevent_cycle!(first_visit_to_tree,
+                                           forbidden_colors,
+                                           v,
+                                           w,
+                                           x,
+                                           g,
+                                           two_colored_forest,
+                                           color)
```

Comment:
did you start from scratch after adding the toml?

Reply:
What does scratch mean? I added the toml and called format on the directory again.
This changed most files and I made another commit.

Reply:
yeah you cannot do that because running it with a different style and then this style does not give the same result as running it only with SciML style.

Reply:
The explanation here gave me the idea that it would be exactly the same end result no matter the starting point.
https://domluna.github.io/JuliaFormatter.jl/stable/how_it_works/#Generating-an-FST

Reply:
Yeah but the formatting isn't commutative, so it actually doesn't work out like that if you have two different styles 😅